### ISSUE TITLE
Wrong indentation warning when using sass maps

### ISSUE THREAD

**Issue by DanGamble89** [stylelint/stylelint#460]
The code: https://gist.github.com/DanGamble89/377cde2725d7e029dfa7

The CLI output:

```
stylelint.scss
15:1	Expected indentation of 2 spaces (indentation)
26:1	Expected indentation of 2 spaces (indentation)
41:1	Expected indentation of 2 spaces (indentation)
65:1	Expected indentation of 2 spaces (indentation)
77:1	Expected indentation of 2 spaces (indentation)
88:1	Expected indentation of 2 spaces (indentation)
95:1	Expected indentation of 2 spaces (indentation)
99:1	Expected indentation of 2 spaces (indentation)
108:1	Expected indentation of 2 spaces (indentation)
116:1	Expected indentation of 2 spaces (indentation)
120:1	Expected indentation of 2 spaces (indentation)
132:1	Expected indentation of 2 spaces (indentation)
139:1	Expected indentation of 2 spaces (indentation)
```

Which seems to expect the closing brace to be indented the specified amount in the config

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/commit/0799c452e6a1d27b34b776598cf1bfc9e7267c12
